cmd/sovereignnode/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/google/gops v0.3.18
 	github.com/multiversx/mx-chain-core-go v1.2.13-0.20230814093628-1c16435b4253
-	github.com/multiversx/mx-chain-go v1.5.13-0.20230901113401-dd2496322a0f
+	github.com/multiversx/mx-chain-go v1.5.13-0.20231004085203-8fda6c5d47ed
```

Comment:
If we update  feat/chain-go-sdk-refactor with feat/chain-go-sdk there is no need to keep updating this since there we have a local replace of the repository(on purpose until the feature is done)

Reply:
will update it right after this merge